editor/src/clj/editor/lsp/async.clj

```diff
@@ -105,5 +105,5 @@
   [ec & body]
   `(let [~ec (g/make-evaluation-context)
          ret# (do ~@body)]
-     (ui/run-later (g/update-cache-from-evaluation-context! ~ec))
+     (fx/on-fx-thread (g/update-cache-from-evaluation-context! ~ec))
```

Comment:
Fix tests failing when transferring new cache entries to the main thread from the LSP worker threads.